pom.xml

```diff
@@ -115,8 +115,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>matrix-project</artifactId>
-            <version>1.4</version>
-            <scope>test</scope>
+            <version>1.7</version>
+            <type>jar</type>
```

Comment:
`type=jar` is the default, need not specify it. Are you intentionally making this into a plugin dependency? I guess because it already was, but implicitly prior to split?